python/smashbox/utilities/__init__.py

```diff
@@ -668,20 +668,24 @@ def check_groups(num_groups=None):
             fatal_check(result, 'Group %s not found' % group_name)
 
 
-def expect_modified(fn, md5):
+def expect_modified(fn, md5, comparedTo=''):
```

Comment:
I get the idea but the argument name 'comparedTo' is not clear as to its intended purpose. Could we, instead, have something very clear like this:

def expect_modified(fn, md5, comment=''):
  ...

and then use it like this:

expect_modified(os.path.join(d,'TEST_FILE_MODIFIED_GROUP_SHARE.dat'), shared['md5_sharer'], comment='compared to original file from sharer")

After all this is just a comment, isn't it?